### ISSUE TITLE
Can't marry a male villager

### ISSUE THREAD

**Issue by Sakurokamizenakura** [Luke100000/minecraft-comes-alive#325]
I haven't found this issue anywhere, so I'm posting it.
I'm playing on forge 1.19 41.1.0, with minecraft comes alive 7.3.14 and architectury api 5.10.33. I found this weird, that either me or my sister (when we play together) can't marry a male villager. For a test, on singleplayer I created world with two males and two females. All of them have 100 hearts with me. I gave the rose gold ring to all of them - both males pulled me off, and females accepted my engagement.
Males says something like "It's not you, it's me! Or... maybe it's us..." or "I'm sorry, but i just don't think we are right for each other", and females "Is it serious? This moment, not ring. Because ring is true?" or "Yes! I'll marry you! I love you!"
![2022-08-03_19 07 16](https://user-images.githubusercontent.com/108942773/182669254-8a2f58cb-5f9b-41fa-b0d3-7c147be8cf19.png)
(I'm playing on polish so I don't know if the translated conversations are perfectly right)
It's the same when I choose the feminine or masculine. I can marry male and female npcs without any issue, it's just the player.
Does someone know how to solve this issue? I don't want to my soon-husband to be my "wife" in ceremony :D

**Comment by CDAGaming** [comment]
The villager needs the `bisexual` trait for marrying the same gender npc. By default, the destiny screen when you first started the game sets the gender as Masculine/Male, so this could perhaps be the issue, and explain why both males said no, while the two female npc's accepted.

**Comment by CDAGaming** [comment]
![image](https://user-images.githubusercontent.com/16328918/182674735-e6aed0bf-568a-4b1c-b585-7fc45c33659b.png)
Test case verified, closed as intended. You can change you gender via the editor command or the destiny screen.

**Comment by Sakurokamizenakura** [comment]
Unfortunately this don't work. As I said, I tested both feminine and masculine. The problem is with both genders. Bisexual traits works, but I feel a little weird, when I try to hug or kiss my mate while having female gender, and always get rejected after removing bisexual trait. When I used the 7.1.0 version this problem didn't existed, everything was okay. Only on this version I have problem don't know why - if I'm female, they reject me, if male - too.
